aten/src/ATen/native/cpu/group_norm_kernel.cpp

```diff
@@ -598,70 +596,111 @@ ComputeInternalGradients(
         ds_vec = ds_vec + dy_fvec1 * x_fvec1;
         db_vec = db_vec + dy_fvec0 + dy_fvec1;
       }
-      ds_vec.store(ds_arr.data());
-      db_vec.store(db_arr.data());
-      opmath_t ds_val = std::accumulate(ds_arr.cbegin(), ds_arr.cend(), float(0));
-      opmath_t db_val = std::accumulate(db_arr.cbegin(), db_arr.cend(), float(0));
-      for (const auto j : c10::irange(inner_size, HxW)) {
-        ds_val += opmath_t(dY_ptr[j]) * opmath_t(X_ptr[j]);
-        db_val += opmath_t(dY_ptr[j]);
+
+      if (j < HxW) {
+        const Vec dy_bvec = Vec::loadu(dY_ptr + j, HxW - j);
+        const Vec x_bvec = Vec::loadu(X_ptr + j, HxW - j);
+        fVec x_fvec0, x_fvec1, dy_fvec0, dy_fvec1;
+        std::tie(x_fvec0, x_fvec1) = convert_to_float<T>(x_bvec);
```

Comment:
May want to replace std::tie with structured bindings here

Reply:
Used `auto [x_fvec0, x_fvec1] =  convert_to_float<T>(x_bvec)` instead.
Thanks for your comments.